service/kas/kas.proto

```diff
@@ -31,6 +31,38 @@ message LegacyPublicKeyRequest {
   string algorithm = 1;
 }
 
+message KeyAccess {
+  string encrypted_metadata = 1;
+  bytes policy_binding = 2;
+  string protocol = 3;
+  string key_type = 4;
+  string kas_url = 5;
+  string kid = 6;
+  string split_id = 7;
+  bytes wrapped_key = 8;
+  bytes header = 9;
+  string algorithm = 10;
```

Comment:
```suggestion
  string algorithm = 10 [json_name = "alg"];
```

Hah this is not currently encoded in the stored KAOs or listed in the spec, so we don't know what to name it. I'd suggest we add it, using `alg` to match the name in the `policyBinding`. We currently can infer it from kid, but that might not always be the case. In practice we will also want several algorithms:

- the one used to encrypt the wrapped key - can be inferred from kid
- to verify the policy binding - explicit in the `policyBinding` object
- to wrap/encapsulate the response - I guess this is up to the server now?
- others? E.g. encryptedMetadata is AES with the unwrappedKey value (which might not be the payload DEK but instead a fragment of the DEK or, eventually, some other value?)

Reply:
I added this within PolicyBinding message